src/hammer-tech/stackup.py

```diff
@@ -225,7 +239,36 @@ def max_width_from_pitch(self, pitch: Decimal) -> Decimal:
         """
         return self.min_spacing_and_max_width_from_pitch(pitch)[1]
 
-    def get_width_spacing_start_twt(self, tracks: int) -> Tuple[Decimal, Decimal, Decimal]:
+    def quantize_to_width_table(self, width: Decimal, layer: str, logger: Optional[HammerVLSILoggingContext]) -> Decimal:
```

Comment:
Could we unit test this method?

Reply:
Did a test using two identical metal objects, one with a width table and one without (just 1 entry of min width). A little klugey but I didn't want to mess up the other existing power strap tests and we probably want to improve the behavior in a future PR anyway to adjust the spacing as well.

Reply:
> we probably want to improve the behavior in a future PR anyway to adjust the spacing as well

👍